### PR TITLE
[importer] adding unit test for direct upload using sourceType as mysql

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- unit test added using pre-exist CSV file (us_population.csv)

### CODE REVIEW

File: desktop/libs/indexer/src/indexer/indexers/sql_tests.py [hunk 1]
```diff
@@ -834,3 +834,36 @@ def test_create_table_from_local():
   `dist` bigint);'''
 
   assert_equal(statement, sql)
+
+
+def test_create_table_from_local_mysql():
+  source = {
+    'path': '/apps/beeswax/data/tables/us_population.csv',
```
Comment (romainr) [inline desktop/libs/indexer/src/indexer/indexers/sql_tests.py:841]: Nice!

Main issue though is that it will tie tests to demo data, and we can't usually test corner cases (i.e. adding a boolean column...) and reusing the same for the other dialects.

For sure we will need to mock the File data with Magic Mock at some point, good for now but to refactor when doing test for hive, impala, phoenix dialects

**Review by romainr — APPROVED** [review]
Nice!

(and later will see for constraints PKs, NOT NULL etc)

**Review by romainr — APPROVED** [review]
Nice!

(and later will see for constraints PKs, NOT NULL etc)
